### PR TITLE
npm initialization

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "leaflet.blurred-location-display",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (mridulnagpal) [inline package.json]: Yeah I am working on it

**Comment by mridulnagpal** [comment]
Have made alot of changes to this PR, not able to solve the test issue though. Also there is an issue with the gh-pages. Can you have a look. Also what do we plan on doing further?

**Comment by jywarren** [comment]
Are all the files in `.grunt` necessary? Maybe take a close look at what files are used in the original library. 

Also it looks like you need to non-capitalize `True` on this line: https://travis-ci.org/publiclab/leaflet-blurred-location-display/jobs/266987755#L563

**Comment by mridulnagpal** [comment]
Still an error!

**Comment by jywarren** [comment]
@publiclab/soc - can anyone working with JavaScript take a look here and help @mridulnagpal with this? Much appreciated!

**Comment by jywarren** [comment]
Hmm, in https://github.com/publiclab/leaflet-blurred-location-display/pull/6/files#diff-4f0907a1b7ae8b13345db1251fd425b2

I think it needs to be in a test block, like:

```js
...
it("converts markdown to html and back", function() {
  expect(...).toBe(...);
})
```
I'm also not sure /all/ of these files are necessary -- are they all included in https://github.com/publiclab/PublicLab.Editor/blob/master/spec/ ??

**Comment by mridulnagpal** [comment]
I think this should work fine, this is like a prototype having basic functionalities for Leaflet.BlurredLocationDisplay. What all other features do I add?

**Comment by mridulnagpal** [comment]
Merging this one for some follow up issues, please let me know if any changes are required.

**Comment by mridulnagpal** [comment]
@jywarren Can you have a look, if its good to merge

**Comment by jywarren** [comment]
🎉

**Pull request merged**